include/onnxruntime/core/graph/constants.h

```diff
@@ -26,5 +26,8 @@ constexpr const char* kNGraphExecutionProvider = "NGRAPHExecutionProvider";
 constexpr const char* kNupharExecutionProvider = "NupharExecutionProvider";
 constexpr const char* kBrainSliceExecutionProvider = "BrainSliceExecutionProvider";
 constexpr const char* kTensorrtExecutionProvider = "TensorrtExecutionProvider";
+
+constexpr int kMaximumAiOnnxVersionSupported = 10;
```

Comment:
shouldn't we be using 
DomainToVersionRange from ONNX
see: https://github.com/onnx/onnx/blob/master/onnx/defs/schema.h#L687
rather than hardcoding it in ORT?

Reply:
Thanks for pointing that out. I like the idea of using it. The issue here (note this fix targets the `rel-1905` Vibranium branch, and so this might not be applicable to the `master` branch) is that we took an older snapshot of ORT that only supported opset 10, but its ONNX submodule has 11. The ORT support in this branch for opset 11 was preliminary and partial though, which was also before the ONNX 1.6 spec was published (meaning ops could still change behavior). So we don't really want 11.

A pending question is what this should ultimately look like in `master`. I'll open a GH issue on it...